RInAppMessaging/Classes/CampaignDispatcher.swift

```diff
@@ -125,4 +141,16 @@ internal class CampaignDispatcher: CampaignDispatcherType, TaskSchedulable {
         return campaignData.intervalBetweenDisplaysInMS ??
             Constants.CampaignMessage.defaultIntervalBetweenDisplaysInMS
     }
+
+    private func data(from url: URL, completion: @escaping (Data?) -> Void) {
+        let sessionConfig = URLSessionConfiguration.default
```

Comment:
Is default cache policy enough in this case or `returnCacheDataElseLoad` (thinking specifically about images) would better? What do you think?

Reply:
Forgot that images could be shown more than once. I think relying on the caching policy defined in the HTTP headers as per the default is fairly sensible. But I guess the main problem is that the default `urlCache` size is something low like 512kB(?).

Reply:
Yeah it's 512kb for memory cache. 
According to this: https://developer.apple.com/documentation/foundation/nsurlsessiondatadelegate/1411612-urlsession
request can't exceed 5% of the allocated disk capacity to be cached. I think we should increase those values to store maybe 4 campaign images (5MB is the limit for campaign image in the dashboard).